pkg/actor/message/message.go

```diff
@@ -34,6 +38,11 @@ type Message struct {
 	Tp Type
 	// BarrierTs
 	BarrierTs model.Ts
+	// Leveldb sorter task
+	// TODO: find a way to hide it behind an interface while saving
+	//       memory allocation.
+	// See https://cs.opensource.google/go/go/+/refs/tags/go1.17.2:src/runtime/iface.go;l=325
+	SorterTask sorter.Task
```

Comment:
Is the interface allocation here a significant overhead?

Reply:
It helps golang runtime GC when we have many message and actors.